api/src/domain/license-status/searchLicenseStatusFactory.test.ts

```diff
@@ -97,13 +111,67 @@ describe("searchLicenseStatus", () => {
     });
 
     const result = await searchLicenseStatus(nameAndAddress, "Home improvement");
-    expect(result.checklistItems).toEqual([
-      {
-        title: "Item 2",
-        status: "ACTIVE",
-      },
+    expect(result.status).toEqual("EXPIRED");
+    expect(result.checklistItems).toEqual(
+      expect.arrayContaining([
+        {
+          title: "Item 1",
+          status: "ACTIVE",
+        },
+        {
+          title: "Item 2",
+          status: "ACTIVE",
+        },
+      ])
+    ); 
+  } );
+
+  it("returns a license if it doesn't have an issue date", async () => {
+    stubLicenseStatusClient.search.mockResolvedValue([
+     generateLicenseEntity({
+        addressLine1: "1234 Main St",
+        applicationNumber: "SOME APPLICATION NUMBER WITHOUT AN ISSUE DATE",
+        checklistItem: "ACTIVE APPLICATION WITHOUT AN ISSUE DATE",
+        issueDate: undefined
+      }), 
+      generateLicenseEntity({
+        addressLine1: "1234 Main St",
+        applicationNumber: "12345",
+        checklistItem: "Item 1",
+        checkoffStatus: "Completed",
+        licenseStatus: "Expired",
+        issueDate: undefined,
+        dateThisStatus: "20200405 000000.000"
+      }),
+      generateLicenseEntity({
+        applicationNumber: "12345",
+        checklistItem: "Item 2",
+        checkoffStatus: "Completed",
+        issueDate: undefined,
+        dateThisStatus: "20200405 000000.000"
```

Comment:
having this entity stubbed seems irrelevant to the test, does it provide extra value or just noise? seems like the test is comparing being able to select "item 1" vs "active application...." right?

although, I'm a bit confused as to why "Item 1" winds up being the one that matches. even though the other does not have an issue Date, presumably it does have another date that it falls back on? The fact that its date is not defined in the test is a bit confusing